en-US/snippets/2017/oct2017.lang

```diff
@@ -0,0 +1,99 @@
+## NOTE: Reference bug 1408280. Deadline: 25 October, 2017.
+## NOTE: Guidelines on localizing snippets: https://goo.gl/nEhdt4. The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: No character length limit, but try not to exceed English length.
+
+
+# Snippet for Firefox quantum
+# DeLorean refers to the time machine from the Back to the Future films. If there are better known time machine reference in your culture, feel free to swap it out.
+# Link: https://www.mozilla.org/firefox/quantum/
+;See the future of Firefox, no DeLorean required. <a>Check out Quantum.</a>
+See the future of Firefox, no DeLorean required. <a>Check out Quantum.</a>
+
+
+# Snippet for Firefox quantum
+# “aka” is the acronym for “also known as”. Feel free to substitute it with the equivalent in your language.
+# Link: https://www.mozilla.org/firefox/quantum/
+;Don’t miss out — Firefox Quantum (aka the best Firefox ever) launches soon. <a>Get notified when it becomes available.</a>
+Don’t miss out — Firefox Quantum (aka the best Firefox ever) launches soon. <a>Get notified when it becomes available.</a>
+
+
+# Snippet for Firefox features
+# The repeat use of the word "fast" is to convey to users that Firefox is the speedest it's ever been.
+;The new Firefox can be summed up in two words: faster and faster. OK, maybe that’s the same word twice.<a>Learn More.</a>
+The new Firefox can be summed up in two words: faster and faster. OK, maybe that’s the same word twice.<a>Learn More.</a>
+
+
+# Snippet for Firefox features
+# The repeat use of the word "fast" is to convey to users that Firefox is the speedest it's ever been.
+;The new Firefox is all about fast. Faster startups. Faster tab switching. Faster scrolling. Faster at letting you get more done. <a>Learn more.</a>
+The new Firefox is all about fast. Faster startups. Faster tab switching. Faster scrolling. Faster at letting you get more done. <a>Learn more.</a>
+
+
+# Snippet for Firefox features
+# Tab hoarding is the practice of having a lot of browser tabs open.
+;<a>Firefox is refreshingly fast,</a> with quicker startup times, fewer crashes and better at making tab hoarders happy (you know who you are.)
+<a>Firefox is refreshingly fast,</a> with quicker startup times, fewer crashes and better at making tab hoarders happy (you know who you are.)
+
+
+# Snippet for Firefox
+;Firefox is the only browser that’s made by a non-profit, so it doesn’t sell access to your every click. #Nowyouknow
+Firefox is the only browser that’s made by a non-profit, so it doesn’t sell access to your every click. #Nowyouknow
```

Comment:
Are they supposed to localize the hashtag?

Reply:
Probably not. We have had this type before and it was left in English. let me make a note of it